README.md

```diff
@@ -2,22 +2,72 @@
 
 [![Test action](https://github.com/LemLib/pros-build/actions/workflows/test.yml/badge.svg)](https://github.com/LemLib/pros-build/actions/workflows/test.yml)
 
-## Usage
+This action creates an environment capable of building PROS projects and templates, and builds them using [build.sh](/build-tools/build.sh)
 
-Here are the basics `steps` you'd need to build a pros project:
+
+Instructions on creating a custom build script, adding additional packages, and using this image as a base are located at the end of this readme.
+
+## Usage:
 
 ```yml
```

Comment:
why maintain this example workflow in two places? its probably best to only have it in the readme